40 search section

cesn4 opened the pull request:
Uzsidejau pora TODO biskuti cia. Tingiu kolkas prie to sedeti :D, norisi page pasibaigti greiciau.

cesn4 committed:
DiscoverySearch section created

cesn4 committed:
InputDropdown created

cesn4 committed:
Mobile view

cesn4 committed:
mobile view fix

cesn4 committed:
Tablet + Desktop views

cesn4 committed:
radius Added

cesn4 committed:
radius was changed

cesn4 committed:
TODO added

cesn4 committed:
Merge branch 'dev' into 40-search-section

RokasAnisas commented on src/sections/DiscoveryHero/DiscoveryHero.scss:
```diff
@@ -0,0 +1,49 @@
+        padding: 90px spacer(small-xl);
+
+        @include if-tablet {
+            padding: spacer(trojan-xs) 170px;
```
`170px`  maciau egzistuoja prie spaceriu

cesn4 replied:
Damn sitie spaceriai pareina is Hero, kai yra jie jau sutvarkyti ten. Hero ismerginau tada dar nepatvarkyta, kad matyciau bendra vaizda matyciau.

RokasAnisas commented on src/sections/DiscoveryHero/DiscoveryHero.scss:
```diff
@@ -0,0 +1,49 @@
+            border-bottom-left-radius: 0;
+            border-bottom-right-radius: radius(l);
+            border-top-right-radius: radius(l);
+            padding: 150px spacer(trojan-xs);
```
`170px` egzistuoja prie spaceriu, 150px gal irgi pridet tada ?

RokasAnisas commented on src/sections/DiscoveryHero/DiscoveryHero.scss:
```diff
@@ -0,0 +1,49 @@
+        @include if-desktop {
+            border-bottom-left-radius: radius(l);
+            border-top-left-radius: radius(l);
+            padding: 150px spacer(trojan-xs);
```
`170px` egzistuoja prie spaceriu, 150px gal irgi pridet tada ?

RokasAnisas commented on src/components/InputDropdown/InputDropdown.scss:
```diff
@@ -0,0 +1,12 @@
+        font-size: font-size("paragraph");
+        font-weight: $font-weight-fat;
+        line-height: $line-height-medium;
+        margin-bottom: 10px;
```
spacer ?

cesn4 committed:
Unit test for Input component

cesn4 committed:
spacer fix

RokasAnisas commented on src/components/DiscoverySearch/DiscoverySearch.scss:
```diff
@@ -0,0 +1,12 @@
+.discovery-search {
+    height: 100%;
```
optional: `size(100%);`

RokasAnisas commented on src/components/DiscoveryTitle/DiscoveryTitle.scss:
```diff
@@ -0,0 +1,31 @@
+    font-family: $font-family-primary;
+    font-weight: $font-weight-fat;
+
+
```
2 empy lines per daug

cesn4 replied:
Sitas irgi dublikuojas :/

RokasAnisas commented on src/components/DiscoverySearch/DiscoverySearch.tsx:
```diff
@@ -0,0 +1,27 @@
+
+import InputDropdown from '~/components/InputDropdown';
+import Input from '~/components/Input';
+import SimpleButton from '../SimpleButton';
```
relative import? Not a child, naudok`~/`

cesn4 committed:
Used function for sizes

cesn4 committed:
import navigation changes

cesn4 committed:
Merge branch 'dev' into 40-search-section

cesn4 committed:
Search position in the parent changed to center options instead of padding position

RokasAnisas commented on src/components/DiscoverySearch/DiscoverySearch.tsx:
```diff
@@ -0,0 +1,27 @@
+
+import './DiscoverySearch.scss';
+
+const DiscoverySearch: FunctionComponent = () => {
```
Unit test

RokasAnisas commented on src/components/InputDropdown/InputDropdown.tsx:
```diff
@@ -0,0 +1,22 @@
+    ];
+    return (
+        <div className={className}>
+            <span className={`${className}__label`}>Select date</span>
```
`InputDropdown` bet ihardcodintas select date, labai jau specifiska :D

RokasAnisas commented on src/components/InputDropdown/InputDropdown.tsx:
```diff
@@ -0,0 +1,22 @@
+        <div className={className}>
+            <span className={`${className}__label`}>Select date</span>
+            {/* TODO change font family */}
+            <Select options={options}></Select>
```
Siulyciau atskirai susikurt `~/components/Select` kuris bus kaip wrapperis sitam `react-select` komponentui, kur ji perstylinsi pagal puslapio dizaina.

cesn4 committed:
Created ReactSelect wrapper

cesn4 committed:
Passed prop to InputDropdown title

cesn4 committed:
Refactured Input components

Visdelto taip padariau, sugrupavau inputus, ir isvedziau propsus. Norejau isvesti list Propsa auksciau, bet Typescriptas labai pyko, kad react-select comopnentas nepriima undetified galimybes ar pns. Kolkas palieku taip, nes ir taip labai daug laiko praleidau kolkas prie sitos problemos, o kadangi dropdowno daugiau nenaudosim tai pasilieku kaip TODO thinga :D

cesn4 committed:
TODO

cesn4 committed:
typo fix

RokasAnisas commented on src/components/InputLabel/InputLabel.tsx:
```diff
@@ -0,0 +1,35 @@
+};
+
+interface InputLabelProps {
+    name: InputType;
```
`InputLabel name=` gal netinkamas propsas nustatyt tipui, `Icon` tiko nes icon gali tiketis ka reiks name. O cia tu turi omeny kazka kito, tipa. Net ir value vadinasi `InputType`, tai siulyciau pervadinti i type.

RokasAnisas commented on src/components/InputLabel/Inputs/DropdownInput/DropdownInput.tsx:
```diff
@@ -0,0 +1,28 @@
+    );
+};
+
+interface DropdownINputProps {
```
DropdownI**N**putProps -> DropdownInputProps

RokasAnisas commented on src/components/InputLabel/Inputs/SimpleInput/SimpleInput.tsx:
```diff
@@ -0,0 +1,48 @@
+import React, { FunctionComponent } from 'react';
+import classNames from 'classnames';
+
+// TODO change png files to svg //
```
Kaip su situo todo ?

cesn4 committed:
typo + name renamed to type

cesn4 committed:
SVGSend component created

cesn4 committed:
Img replaced with svg

RokasAnisas commented on src/components/DiscoverySearch/DiscoverySearch.tsx:
```diff
@@ -0,0 +1,26 @@
+                <InputLabel label="Enter the first keywords" type="basic" />
+            </div>
+            <div className={`${className}__button`}>
+                {/* TODO re-do hover */}
```
Ka su situo darom ?

cesn4 replied:
Pasikursiu taska kur improvinsiu visus hoverius i guess, nes ten ir dropdownui reikia refaktorinimo, next story irgi. Kolkas noriu kad veiktu viskas kaip priklauso :D

RokasAnisas commented on src/components/InputLabel/Inputs/DropdownInput/DropdownInput.tsx:
```diff
@@ -0,0 +1,28 @@
+    label,
+}: DropdownInputProps) => {
+    const className = 'dropdown-input';
+    // TODO make a props for this one ( react-select doesnt accept undefined )
```
Ka su situo darom ?

cesn4 replied:
Kai ka sugalvojau tuoj pameginsiu :D

cesn4 committed:
Prop created for React-List / Backup error added

cesn4 committed:
BackStop paused

cesn4 committed:
Merge branch 'dev' into 40-search-section
